Reference/C.1.0.bpmn

```diff
@@ -387,20 +546,30 @@
         <signavio:signavioMetaData metaKey="risiko" metaValue=""/>
         <signavio:signavioMetaData metaKey="erteiltfreigabe" metaValue=""/>
         <signavio:signavioMetaData metaKey="riskandcontrols" metaValue=""/>
+        <w4graph:graphStyle>
+          <w4graph:basic background="194,215,235" foreground="0,0,0" autoResize="false" borderColor="100,100,100" collapsed="false"/>
+        </w4graph:graphStyle>
       </extensionElements>
       <incoming>invoiceApproved</incoming>
       <outgoing>SequenceFlow_2</outgoing>
+      <potentialOwner id="Bpmn_ResourceRole_qEZisJ1_EeS1-pEyeWEPig" name="Prepare Bank Transfer Potential Owner">
+        <resourceRef>Bpmn_Resource_XNpfIJ1_EeS1-pEyeWEPig</resourceRef>
+      </potentialOwner>
     </userTask>
     <endEvent id="invoiceProcessed" name="Invoice&#xA;processed">
       <extensionElements>
         <signavio:signavioMetaData metaKey="bgcolor" metaValue="#ffffff"/>
         <signavio:signavioMetaData metaKey="userstory" metaValue="[]"/>
+        <w4graph:graphStyle>
+          <w4graph:basic background="190,0,0" foreground="0,0,0" autoResize="false" borderColor="100,100,100" collapsed="false"/>
+        </w4graph:graphStyle>
       </extensionElements>
       <incoming>SequenceFlow_3</incoming>
     </endEvent>
-    <serviceTask id="archiveInvoice" camunda:delegateExpression="#{archiveService}" name="Archive&#xA;Invoice" implementation="webService">
+    <serviceTask id="archiveInvoice" camunda:delegateExpression="#{archiveService}" name="Archive&#xA;Invoice" completionQuantity="1" isForCompensation="false" startQuantity="1" implementation="webService">
       <extensionElements>
         <camunda:field name="text0" stringValue="Hello World"/>
+        <camunda:field name="text1"><![CDATA[
```

Comment:
@benmichel I'll remove the line manually to get the issue closed, but please fix your tool as well.